app/src/main/java/com/quran/labs/androidquran/util/QuranFileUtils.java

```diff
@@ -355,8 +355,21 @@ public static String getQuranAyahDatabaseDirectory(Context context) {
 
   @Nullable
   public static String getQuranAudioDirectory(Context context){
-    String s = QuranFileUtils.getQuranBaseDirectory(context);
-    return (s == null)? null : s + AUDIO_DIRECTORY + File.separator;
+    String path = getQuranBaseDirectory(context);
+    if (path == null) {
+      return null;
+    }
+    path += AUDIO_DIRECTORY;
+    File dir = new File(path);
+    if (dir.exists() && !dir.isDirectory() && !dir.delete()) {
```

Comment:
yes, let's not worry about it since the only way is if someone manually did that manually, and if they did, it's not our fault.

on the same note though, just so i am 100% sure - this writes `.nomedia` in `audio/` - will the various media players know that if there's `.nomedia` in audio, that there doesn't need to be one in `audio/{qari}`?